pynetstring.py

```diff
@@ -86,23 +85,55 @@ def feed(self, data):
                     # Entire buffer was scanned, but no complete length was read
                     break
             if self._state == State.PARSE_DATA:
-                data_end = self._data_offset + self._length
-                if len(self._buffer) < data_end + 1:
-                    # The entire data part including trailing comma has not yet
-                    # been received
-                    break
+                if not self._length:
```

Comment:
Should be self._length == 0 since numerical comparison is intended and not check for uninitialized.

Reply:
Both fit because
```
>>> bool(not None)
True
>>> bool(not 0)
True
```
but I agree `self._length == 0` is more explicit. I'll push new commit.